src/main/java/com/icthh/xm/ms/entity/web/rest/FunctionResource.java

```diff
@@ -129,13 +129,28 @@ public ResponseEntity<Object> callDeleteFunction(@PathVariable("functionKey") St
      */
     @Timed
     @PostMapping("/functions/anonymous/{functionKey:.+}")
-    @PrivilegeDescription("Privilege to execute a function by key (key in entity specification)")
-    public ResponseEntity<Object> callAnonymousFunction(@PathVariable("functionKey") String functionKey,
+    public ResponseEntity<Object> callPostAnonymousFunction(@PathVariable("functionKey") String functionKey,
                                                @RequestBody(required = false) Map<String, Object> functionInput) {
-        FunctionContext result = functionService.executeAnonymous(functionKey, functionInput);
+        FunctionContext result = functionService.executeAnonymous(functionKey, functionInput, "POST");
```

Comment:
better to use constant from spring

Reply:
@sergeysenja1992 
in this case it will look like 
```
HttpMethod.POST.name()
```
because I do not want provide spring constants to LEP context

will it be OK?